documentai/api/DocumentAI.Samples.Tests/DocumentAIFixture.cs

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.DocumentAI.V1;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using System.Threading;
+using Xunit;
+
+[CollectionDefinition(nameof(DocumentAIFixture))]
+public class DocumentAIFixture : IDisposable, ICollectionFixture<DocumentAIFixture>
+{
+    public string ProjectId { get; }
+    public ProjectName ProjectName { get; }
+
+    public string LocationId { get; }
+    public LocationName LocationName { get; }
+
+    public string ProcessorId { get; }
+    public ProcessorName ProcessorName { get; }
+
+    public string LocalPath { get; }
+
+    public string MimeType { get; }
+
+    public DocumentAIFixture()
+    {
+        ProjectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        if (string.IsNullOrEmpty(ProjectId))
+        {
+            throw new Exception("missing GOOGLE_PROJECT_ID");
+        }
+        ProjectName = new ProjectName(ProjectId);
+
+        LocationId = "us";
+        LocationName = new LocationName(ProjectId, LocationId);
+
+        ProcessorId = "8219a5b956a3e0d8";
```

Comment:
Could you create a new processor on the fixture, instead of havin a fixed one?
Does it take too long? And then we'd need to delete it on tear down.

If not, it is better if we store this value in an env variable. Let me know and I can do that.

Reply:
OK, let's use DOCUMENT_AI_PROCESSOR_ID, I've already added it.

Reply:
Ok, the test seems to have failed because the Env variable is missing